pyscf/adc/uadc.py

```diff
@@ -299,24 +304,26 @@ def compute_amplitudes(myadc, eris):
         del (temp)
         del (temp_1)
 
-        t2_2_temp = None
         if isinstance(eris.VVVV_p, np.ndarray):
             eris_VVVV = eris.VVVV_p
             temp = np.ascontiguousarray(t2_1_b[:,:,ab_ind_b[0],ab_ind_b[1]]).reshape(nocc_b*nocc_b,-1)
-            t2_2_temp = np.dot(temp,eris_VVVV.T).reshape(nocc_b,nocc_b,-1)
+            t2_1_vvvv_b = np.dot(eris_VVVV,temp.T)
```

Comment:
Since you're transposing in 311, you can avoid that transpose by evaluating t2_1_vvvv_b as np.dot(temp,eris_VVVV.T). Please address this in all similar contractions in UADC and RADC.